src/components/notifications/HdNotificationsBar.vue

```diff
@@ -96,76 +74,46 @@ export default {
 @import 'homeday-blocks/src/styles/mixins.scss';
 
 .notifications-bar {
-  position: absolute;
-  top: 0;
-  right: 0;
-  left: 0;
   display: flex;
-  justify-content: center;
-  opacity: 0;
   padding: $stack-m $inline-s;
-  transition: opacity ($time-s * 2) ease-in-out;
-  z-index: 10;
-
-  &--visible {
-    opacity: 1;
-  }
-
-  &--compact {
-    padding: $stack-s $inline-s;
-  }
-
-  &--enter-active,
-  &--leave-active {
-    transition: all ($time-s * 2) ease-in-out;
-  }
-
-  &--enter,
-  &--leave-to {
-    transform: translateY(-100%);
-    opacity: 0;
-  }
+  color: $white;
 
   &--notification {
-    background-color: $tertiary-color;
-    color: $white;
+    background-color: $crusta-orange;
   }
 
   &--success {
-    background-color:$success-color;
-    color: $white;
+    background-color: $success-color;
   }
 
   &--error {
-    background-color:#E00016;
-    color: $white;
+    background-color: $monza-red;
   }
 
-  &--warning {
-    background-color:#FFAB00;
-    color: $primary-color;
+  &--info {
+    background-color: getShade($cello-blue, 80);
   }
 
   &__icon {
-    align-items: center;
-    display: flex;
+    width: 24px;
     height: 24px;
-    justify-content: center;
+    min-width: 24px;
+    min-height: 24px;
```

Comment:
Oh, you are totally right =D
I've added the min attributes because before the SVG was inside of a div, and without adding a `min-width/height` the icon was squashed depending on the size of the text.

But in this case, I really don't need to add them.

Thanks =D

Here are the changes:
f6f2b5f